src/lib/webServer.ts

```diff
@@ -36,49 +47,69 @@ export class WebServer {
             this.server = http.createServer(this.app);
             return this.server;
         }
+        const config: AdapterConfig = this.adapter.config as AdapterConfig;
```

Comment:
if you pass an attribute with as to a const, you don't need to specify the type again